Note: gnat_ug.texi is postprocessed by some tool - so it may not
contain tab characters - spaces must be used instead.
In other *.texi we can use tabs.

--- gcc-3.2/gcc/doc/cpp.texi.orig	Fri Nov  1 22:58:48 2002
+++ gcc-3.2/gcc/doc/cpp.texi	Fri Nov  1 23:03:38 2002
@@ -53,9 +53,9 @@
 @set cppmanual
 
 @ifinfo
-@dircategory Programming
+@dircategory Programming Languages:
 @direntry
-* Cpp: (cpp).		       The GNU C preprocessor.
+* Cpp: (cpp).				The GNU C preprocessor
 @end direntry
 @end ifinfo
 
--- gcc-3.2/gcc/doc/cppinternals.texi.orig	Mon Jan  7 20:03:36 2002
+++ gcc-3.2/gcc/doc/cppinternals.texi	Fri Nov  1 23:04:42 2002
@@ -3,9 +3,9 @@
 @settitle The GNU C Preprocessor Internals
 
 @ifinfo
-@dircategory Programming
+@dircategory Programming Languages:
 @direntry
-* Cpplib: (cppinternals).      Cpplib internals.
+* Cpplib: (cppinternals).		Cpplib internals
 @end direntry
 @end ifinfo
 
--- gcc-3.2/gcc/doc/gcc.texi.orig	Fri Nov  1 22:58:48 2002
+++ gcc-3.2/gcc/doc/gcc.texi	Fri Nov  1 23:06:12 2002
@@ -65,9 +65,9 @@
 1999, 2000, 2001, 2002 Free Software Foundation, Inc.
 @end macro
 @ifnottex
-@dircategory Programming
+@dircategory Programming Languages:
 @direntry
-* gcc: (gcc).                  The GNU Compiler Collection.
+* gcc: (gcc).				The GNU Compiler Collection
 @end direntry
 This file documents the use of the GNU compilers.
 @sp 1
--- gcc-3.2/gcc/doc/gccint.texi.orig	Wed Jan 23 18:30:28 2002
+++ gcc-3.2/gcc/doc/gccint.texi	Fri Nov  1 23:06:57 2002
@@ -51,9 +51,9 @@
 1999, 2000, 2001, 2002 Free Software Foundation, Inc.
 @end macro
 @ifnottex
-@dircategory Programming
+@dircategory Programming Languages:
 @direntry
-* gccint: (gccint).            Internals of the GNU Compiler Collection.
+* gccint: (gccint).			Internals of the GNU Compiler Collection
 @end direntry
 This file documents the internals of the GNU compilers.
 @sp 1
--- gcc-3.2/gcc/ada/gnat-style.texi.orig	Sun Dec 23 12:23:14 2001
+++ gcc-3.2/gcc/ada/gnat-style.texi	Fri Nov  1 23:08:50 2002
@@ -25,9 +25,9 @@
 
 @setfilename gnat-style.info
 @settitle GNAT Coding Style
-@dircategory Programming
+@dircategory Programming Languages:
 @direntry 
-* gnat-style: (gnat-style).      GNAT Coding Style
+* gnat-style: (gnat-style).		GNAT Coding Style
 @end direntry
 
 @macro syntax{element}
--- gcc-3.3/gcc/ada/gnat_rm.texi.orig	2003-02-04 02:55:38.000000000 +0100
+++ gcc-3.3/gcc/ada/gnat_rm.texi	2003-05-17 08:40:07.000000000 +0200
@@ -24,9 +24,9 @@
 
 @include gcc-common.texi
 
-@dircategory GNU Ada tools
+@dircategory Programming Languages:
 @direntry
-* GNAT Reference Manual: (gnat_rm).  Reference Manual for GNU Ada tools.
+* GNAT Reference Manual: (gnat_rm).	Reference Manual for GNU Ada tools
 @end direntry
 
 @copying
--- gcc-3.2/gcc/ada/gnat_ug.texi.orig	Fri Nov  1 22:58:52 2002
+++ gcc-3.2/gcc/ada/gnat_ug.texi	Fri Nov  1 23:10:20 2002
@@ -75,6 +75,11 @@
 @syncodeindex fn cp
 @c %**end of header
 
+@dircategory Programming Languages:
+@direntry
+* gnat_ug: (gnat_ug).                   GNAT User's Guide
+@end direntry
+
 @titlepage
 
 @ifset vms
--- gcc-3.2/gcc/f/g77.texi.orig	Mon Apr 29 09:58:32 2002
+++ gcc-3.2/gcc/f/g77.texi	Fri Nov  1 23:11:56 2002
@@ -90,9 +90,9 @@
 @c @end tex
 
 @ifinfo
-@dircategory Programming
+@dircategory Programming Languages:
 @direntry
-* g77: (g77).                  The GNU Fortran compiler.
+* g77: (g77).				The GNU Fortran compiler
 @end direntry
 @ifset INTERNALS
 @ifset USING
--- gcc-3.2/gcc/java/gcj.texi.orig	Sun Aug  4 18:55:55 2002
+++ gcc-3.2/gcc/java/gcj.texi	Fri Nov  1 23:16:39 2002
@@ -20,26 +20,20 @@
 
 @ifinfo
 @format
-@dircategory Programming
+@dircategory Programming Languages:
 @direntry
-* Gcj: (gcj).               Ahead-of-time compiler for the Java language
+* Gcj: (gcj).				Ahead-of-time compiler for the Java language
 @end direntry
 
-@dircategory Individual utilities
+@dircategory Programming tools:
 @direntry
-* gcjh: (gcj)Invoking gcjh.
-                            Generate header files from Java class files
-* jv-scan: (gcj)Invoking jv-scan.
-                            Print information about Java source files
-* jcf-dump: (gcj)Invoking jcf-dump.
-                            Print information about Java class files
-* gij: (gcj)Invoking gij.   GNU interpreter for Java bytecode
-* jv-convert: (gcj)Invoking jv-convert.
-                            Convert file from one encoding to another
-* rmic: (gcj)Invoking rmic.
-                            Generate stubs for Remote Method Invocation.
-* rmiregistry: (gcj)Invoking rmiregistry.
-                            The remote object registry.
+* gcjh: (gcj)gcjh.			Generate header files from Java class files
+* jv-scan: (gcj)jv-scan.		Print information about Java source files
+* jcf-dump: (gcj)jcf-dump.		Print information about Java class files
+* gij: (gcj)gij.			GNU interpreter for Java bytecode
+* jv-convert: (gcj)jv-convert.		Convert file from one encoding to another
+* rmic: (gcj)rmic.			Generate stubs for Remote Method Invocation.
+* rmiregistry: (gcj)rmiregistry.	The remote object registry
 @end direntry
 @end format
 
@@ -120,15 +114,15 @@
 * Copying::		 The GNU General Public License
 * GNU Free Documentation License::
 			How you can share and copy this manual
-* Invoking gcj::	Compiler options supported by @command{gcj}
+* gcj::	                Compiler options supported by @command{gcj}
 * Compatibility::	Compatibility between gcj and other tools for Java
-* Invoking gcjh::       Generate header files from class files
-* Invoking jv-scan::    Print information about source files
-* Invoking jcf-dump::   Print information about class files
-* Invoking gij::	Interpreting Java bytecodes
-* Invoking jv-convert:: Converting from one encoding to another
-* Invoking rmic::        Generate stubs for Remote Method Invocation.
-* Invoking rmiregistry:: The remote object registry.
+* gcjh::                Generate header files from class files
+* jv-scan::             Print information about source files
+* jcf-dump::            Print information about class files
+* gij::	                Interpreting Java bytecodes
+* jv-convert::          Converting from one encoding to another
+* rmic::                Generate stubs for Remote Method Invocation.
+* rmiregistry::         The remote object registry.
 * About CNI::           Description of the Cygnus Native Interface
 * Resources::		Where to look for more information
 @end menu
@@ -139,7 +133,7 @@
 @include fdl.texi
 
 
-@node Invoking gcj
+@node gcj
 @chapter Invoking gcj
 
 @c man title gcj Ahead-of-time compiler for the Java language
@@ -506,7 +500,7 @@
 @end itemize
 
 
-@node Invoking gcjh
+@node gcjh
 @chapter Invoking gcjh
 
 @c man title gcjh generate header files from Java class files
@@ -607,7 +601,7 @@
 
 @c man end
 
-@node Invoking jv-scan
+@node jv-scan
 @chapter Invoking jv-scan
 
 @c man title jv-scan print information about Java source file
@@ -665,7 +659,7 @@
 
 @c man end
 
-@node Invoking jcf-dump
+@node jcf-dump
 @chapter Invoking jcf-dump
 
 @c man title jcf-dump print information about Java class files
@@ -720,7 +714,7 @@
 
 @c man end
 
-@node Invoking gij
+@node gij
 @chapter Invoking gij
 
 @c man title gij GNU interpreter for Java bytecode
@@ -798,7 +792,7 @@
 
 @c man end
 
-@node Invoking jv-convert
+@node jv-convert
 @chapter Invoking jv-convert
 
 @c man title jv-convert Convert file from one encoding to another
@@ -857,7 +851,7 @@
 
 @c man end
 
-@node Invoking rmic
+@node rmic
 @chapter Invoking rmic
 
 @c man title rmic Generate stubs for Remote Method Invocation
@@ -936,7 +930,7 @@
 @c man end
 
 
-@node Invoking rmiregistry
+@node rmiregistry
 @chapter Invoking rmiregistry
 
 @c man title rmiregistry Remote object registry
--- gcc-3.2/gcc/ksi/doc/ksi.texi.orig	Sat Jul 20 22:48:08 2002
+++ gcc-3.2/gcc/ksi/doc/ksi.texi	Fri Nov  1 23:19:39 2002
@@ -15,7 +15,7 @@
 @contents
 
 @ifnottex
-@dircategory Programming tools:
+@dircategory Programming Languages:
 @direntry
 * ksi: (ksi).				Ksi Language and Compiler Manual
 @end direntry
